backend/tracim_backend/migration/versions/32e629b17e2e_change_calendar_enabled_to_agenda_enabled_colum.py

```diff
@@ -5,20 +5,21 @@
 Create Date: 2019-04-04 17:39:54.241996
 
 """
+# revision identifiers, used by Alembic.
 from alembic import op
+from sqlalchemy import Boolean
 
-# revision identifiers, used by Alembic.
 revision = "32e629b17e2e"
 down_revision = "f889c2b59759"
 
 
 def upgrade():
     with op.batch_alter_table("workspaces") as bop:
-        bop.alter_column("calendar_enabled", new_column_name="agenda_enabled")
+        bop.alter_column("calendar_enabled", new_column_name="agenda_enabled", type_=Boolean)
```

Comment:
migrations are not supposed to change. What is the reason for this change?

Reply:
i agree. This was needed to fix issue during test of migration for mysql. (this migration is the only one to fail). I do not really know why this parameter "type_" is now needed whereas it was not at all needed before. I can inspect deeper if you think that useful. Anyway this fix, solve the issue.

Reply:
what are we supposed to do on our production? migrate down and up again? Manually patch the database?

Reply:
My feeling is downgrade/upgrade is ok, but I'd like to be sure.

Reply:
> what are we supposed to do on our production? migrate down and up again? Manually patch the database?
Nothing. old behaviour does exactly the same. It's just that for unknown reason it's now necessary to specify type_ for mysql. If you want to be sure, i can verify column types from old and new code to exactly match, but i do no see any reason that this method convert column type to another one when we don't specify it (+ if it does something like that, it should probably make many other test failed)

Reply:
ok, thanks for the explanation